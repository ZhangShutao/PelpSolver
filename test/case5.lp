f :- p.
f :- q.
p | q.
:- K[0, 1) q, q.
:- K[0, 1) p, p.
:- K[0, 1) f.