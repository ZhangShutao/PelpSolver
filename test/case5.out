WorldView 1:
{[1.0]:{q,f}}
WorldView 2:
{[1.0]:{p,f}}